internal/build.go

```diff
@@ -124,7 +124,7 @@ func finalizeEnvelope(ctx context.Context, env *gobl.Envelope, opts *BuildOption
 	if len(env.Signatures) > 0 {
 		return echo.NewHTTPError(http.StatusConflict, "document has already been signed")
 	}
-	if err := env.Complete(); err != nil {
+	if err := env.Calculate(); err != nil {
```

Comment:
Nice! This makes a lot more sense now semantically. 👍

Reply:
This func `finalizeEnvelope` is still a bit generically named, but I think that's fine, because it's small enough to grok what it does at a glance.

Reply:
Agreed, we can refactor a few of these with the new commands.